README.md

```diff
@@ -922,6 +922,7 @@ Table of Contents
   * [ddos-guard.net](https://ddos-guard.net/store/web) — Free CDN, DDoS protection and SSL certificate
   * [developers.google.com](https://developers.google.com/speed/libraries/) — The Google Hosted Libraries is a content distribution network for the most popular, Open Source JavaScript libraries
   * [GraphCDN](https://graphcdn.io/) - Scale, inspect & protect your GraphQL API. 5m requests per month for free.
+  * [Jotcode](https://jotcode.io) - A serverless platform for hosting simple scripts, API and servers
```

Comment:
Please include someting like `free plan allows running 100 scripts per month`

Reply:
Sure. I just did it :) Thanks